tools/erofs-utils/Makefile

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (C) 2009-2025 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=erofs-utils
+PKG_VERSION:=1.8.9
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://git.kernel.org/pub/scm/linux/kernel/git/xiang/erofs-utils.git
+PKG_MIRROR_HASH:=feab7386de6faf11cb29af5bfa240ea119b14bfd66c14d80de0509c1ab16dcc6
+PKG_SOURCE_DATE:=2025-06-26
+PKG_SOURCE_VERSION:=81169bf3cfd26b8f2b3aa3b20da23971168a90a9
+
+PKG_FIXUP:=autoreconf
+
+include $(INCLUDE_DIR)/host-build.mk
+
+HOST_CONFIGURE_ARGS=\
+	--enable-multithreading \
+	--enable-lzma \
+	--enable-lz4 \
```

Comment:
Why not also enable ZSTD support since we build it anyway?

Reply:
I don't want enable zstd for now until new erofs-utils 1.9. zstd is still experimental in v1.8 and I think in the image size it's uncomparable to lzma.